unison-src/transcripts/api-namespace-list.output.md

```diff
@@ -71,7 +71,8 @@ GET /api/list?namespace=nested.names
         {
             "contents": {
                 "namespaceHash": "#n1egracfeljprftoktbjcase2hs4f4p8idbhs5ujipl42agld1810hrq9t7p7ped16aagni2cm1fjcjhho770jh80ipthhmg0cnsur0",
-                "namespaceName": "x"
+                "namespaceName": "x",
+                "numContainedDefinitions": null
```

Comment:
These are expected to be null on local-share since it doesn't have a name-lookup-index.